json/jenkins_job_categories.json

```diff
@@ -0,0 +1,100 @@
+{
+    "dashboard": {
+        "id": "5c707260-2a0f-11e9-b1b8-f3720d7eacea",
+        "value": {
+            "description": "",
```

Comment:
Maybe it is not very important, but I miss a brief description here. We could use the same description from the Markdown file.